src/cloudformation/vpc.template

```diff
@@ -517,11 +477,25 @@
         "Tags": [
           {
             "Key": "Name",
-            "Value": {"Fn::Join": ["-", [{"Ref": "AWS::StackName"}, "nat"]]},
+            "Value": {"Fn::Join": ["-", [{"Ref": "AWS::StackName"}, "bastion"]]},
             "PropagateAtLaunch": true
           }
         ]
       }
+    },
+    "NatEip": {
```

Comment:
Future-proofing: `NatEip01`? Otherwise if we want to add `NatEip0x` later this EIP will be replaced